client/implementation/src/test/java/test/unit/TypeInfoBehaviour.java

```diff
@@ -0,0 +1,27 @@
+package test.unit;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import io.smallrye.graphql.client.typesafe.impl.reflection.TypeInfo;
+
+class TypeInfoBehaviour {
```

Comment:
This has to be the American spelling `Behavior`, or it won't be picked up by the config in the `maven-surefire-plugin`.